bftengine/src/bftengine/SigManager.cpp

```diff
@@ -237,22 +251,21 @@ bool SigManager::verifySig(
   return result;
 }
 
-void SigManager::sign(const char* data, size_t dataLength, char* outSig, uint16_t outSigLength) const {
+void SigManager::sign(const char* data, size_t dataLength, char* outSig) const {
```

Comment:
My previous comment is missing for this api i think. This interface is not very clear to me. There is an assumption here that outSig points to a valid memory to hold sig data. We should not make such an assumption. We should either have an outSigLen parameter or use std::array for outSig. Also, necessary description in the header file for this api

Reply:
I agree with you. I deleted `outSigLength` because it was unused, and also it was confusing. `outSig` always points to a valid memory location. So, I found `outSigLength` existence not so useful. Also, I thought it would be best to keep the other implementation as is (ofcourse after deleting `outSigLength`). wdyt?